cookbook/recipes/core/intermediate/typed_schema.py

```diff
@@ -19,8 +19,8 @@
 
 
 # %%
-# To write to a schema object refer to :py:func:`flytekit.FlyteSchema.open` method. Writing can be done using
-# Any of the supported dataframe formats.
+# To write to a schema object refer to :py:method:`flytekit.types.schema.FlyteSchema.open` method. Writing can be done
```

Comment:
Awesome, eventually I want to use the hovertext plugin to show a modal like source graph